Sources/Networking/AlamofireNetworkClient/AlamofireNetworkClient.swift

```diff
@@ -62,6 +62,7 @@ public extension AlamofireNetworkClient {
     return .init(with: request)
   }
   
+  @available(*, deprecated, message: "Use request method with `encoder` instead.")
   func request<E: Encodable>(
     method: HTTPMethod,
     endpoint: URLConvertible,
```

Comment:
Any objections to deprecating this method?

Reply:
Actually, I need to delete this method since it has the same signature as the new method. Therefore compiler cannot tell which one to pick.